Source/WebKit/UIProcess/API/glib/WebKitWebsiteDataManager.cpp

```diff
@@ -479,6 +490,41 @@ static void webkit_website_data_manager_class_init(WebKitWebsiteDataManagerClass
             nullptr, nullptr,
             FALSE,
             static_cast<GParamFlags>(WEBKIT_PARAM_READWRITE | G_PARAM_CONSTRUCT_ONLY)));
+
+    /**
+     * WebKitWebsiteDataManager:origin-storage-ratio:
+     *
+     * The ratio which is used to calculate the origin data storage based on disk space.
+     * The origin data storage if it is exceeded, request will fail.
+     * Value of 0.0 means that data storage is not allowed.
```

Comment:
So, if I understand this correctly, this is the percentage of the disk space (actually the partition/volume) that could be used to store data for every origin, right? I find the documentation a bit confusing.